libtuf-server/src/main/scala/com/advancedtelematic/libtuf_server/http/ServiceHttpClient.scala

```diff
@@ -18,15 +18,15 @@ import cats.syntax.option._
 import scala.concurrent.Future
 import scala.reflect.ClassTag
 
-
+@deprecated("use data type from libats-http", "v0.4.0-40")
 trait ServiceHttpClientSupport {
```

Comment:
Message could be better, this is not a data type

Reply:
Ah this is actually for the other PR, right? I am not sure what you mean. A Http client should go into libats-http on the face of it, message deals with Kafka, no?

Reply:
yeah the http client should go in libats-http, but I meant the deprecated message, it says "data type" but `ServiceHttpClientSupport` is not a data type.

But anyway this should just go and use the one from libats-http.

Reply:
Ah, gotcha. I had assumed we'd keep old stuff as deprecated for a while, but I guess it's not really important here.